src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/SecurityController.php

```diff
@@ -0,0 +1,307 @@
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Controller\Admin\Configure\AdvancedParameters;
+
+use Exception;
+use PrestaShop\PrestaShop\Core\Domain\Security\Command\BulkDeleteCustomerSessionsCommand;
+use PrestaShop\PrestaShop\Core\Domain\Security\Command\BulkDeleteEmployeeSessionsCommand;
+use PrestaShop\PrestaShop\Core\Domain\Security\Command\DeleteCustomerSessionCommand;
+use PrestaShop\PrestaShop\Core\Domain\Security\Command\DeleteEmployeeSessionCommand;
+use PrestaShop\PrestaShop\Core\Domain\Security\Exception\SessionNotFoundException;
+use PrestaShop\PrestaShop\Core\Exception\CoreException;
+use PrestaShop\PrestaShop\Core\Form\FormHandlerInterface;
+use PrestaShop\PrestaShop\Core\Search\Filters\Security\Session\CustomerFilters;
+use PrestaShop\PrestaShop\Core\Search\Filters\Security\Session\EmployeeFilters;
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class SecurityController is responsible for displaying the
+ * "Configure > Advanced parameters > Security" page.
+ */
+class SecurityController extends FrameworkBundleAdminController
+{
+    /**
+     * Show sessions listing page.
+     *
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @return Response
+     */
+    public function indexAction(): Response
+    {
+        $generalForm = $this->getGeneralFormHandler()->getForm();
+
+        return $this->render(
+            '@PrestaShop/Admin/Configure/AdvancedParameters/Security/index.html.twig',
+            [
+                'layoutHeaderToolbarBtn' => [],
+                'layoutTitle' => $this->trans('Security', 'Admin.Navigation.Menu'),
+                'generalForm' => $generalForm->createView(),
+            ]
+        );
+    }
+
+    /**
+     * Process the Security general configuration form.
+     *
+     * @AdminSecurity("is_granted(['update', 'create', 'delete'], request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function processGeneralFormAction(Request $request): RedirectResponse
+    {
+        return $this->processForm(
+            $request,
+            $this->getGeneralFormHandler(),
+            'General'
+        );
+    }
+
+    /**
+     * Process the Security configuration form.
+     *
+     * @param Request $request
+     * @param FormHandlerInterface $formHandler
+     * @param string $hookName
+     *
+     * @return RedirectResponse
+     */
+    protected function processForm(Request $request, FormHandlerInterface $formHandler, string $hookName): RedirectResponse
```

Comment:
I'm surprised we didn't put this into an abstract class yet Almost every configuration controllers have this same method that only varies with the hook base name I guess some refacto/improvement would be possible (not in this PR of course)

Reply:
@jolelievre Good idea for a new PR. May be could you create a new issue for that ?

Reply:
https://github.com/PrestaShop/PrestaShop/issues/23393